### PR TITLE
213 Differentiate broker services and standardize

### PR DESCRIPTION
**Pull request by tom-rm-meyer-ISST**
Handling of the term "metadata broker":
- if "broker" only refer to metadata brokers, name it like that
- if "broker" means multiple types of brokers (app, service, metadata
- see 3.1.1. Roles in the IDS), name it "any broker", "broker service providers" or similar
- if "data broker", name it "metadata broker"

Note: Caps have been ignored (metadata broker, Metadata Broker)

### CODE REVIEW

**Comment by tom-rm-meyer-ISST** [comment]
Proposal to solve https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/issues/213

File: documentation/3_Layers_of_the_Reference_Architecture_Model/3_2_Functional_Layer/3_2_FunctionalLayer.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ is given in the [Security Perspective](../../4_Perspectives_of_the_Reference_Arc
 
 #### Trustworthy Communication & Security by Design ####
 
-Connectors, App Stores, and Brokers can check if the Connector of the
+Connectors, App Stores, and any Broker can check if the Connector of the
```
Comment (sebbader-sap) [inline documentation/3_Layers_of_the_Reference_Architecture_Model/3_2_Functional_Layer/3_2_FunctionalLayer.md:85]: ...any Metadata Broker...

File: documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/4_2_4_Component_Certification.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ Within the next two subsections, three different levels of assurance and trust f
 #### Component Assurance Levels: ####
 
 The depth and rigor of a component evaluation consists of the following three assurance levels,
-independent on the type of component that is being certified (e.g. Connector, Broker, etc.):
+independent on the type of component that is being certified (e.g. Connector, Brokers, etc.):
```
Comment (sebbader-sap) [inline documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/4_2_4_Component_Certification.md:10]: ...Metadata Brokers...

**Review by sebbader-sap — APPROVED** [review]
We need to agree on a uniform spelling. "Metadata Broker" vs. "metadata broker" eliminates already a huge number of previous variations :-)
Proposed updates are fine with me.

**Comment by tom-rm-meyer-ISST** [comment]
> We need to agree on a uniform spelling. "Metadata Broker" vs. "metadata broker" eliminates already a huge number of previous variations :-) Proposed updates are fine with me.

That's a good point I was thinking about, too. Sometimes the text refers to components as names - therefore they were written with starting capitals (such as "Metadata Broker", "Participant Information System"). Sometimes they weren't. I just sticked to the respective chapter (well, they're written by different persons).

As a consequence, I would align on calling it Metadata Broker in uppercase. 

In an additional issue, it could make sense to introduce a naming convention, couldn't it? Something like refering to names,  concepts and entities of the IDS RAM as upper case (like Contract Offer, Usage Policy) and add something like a preamble to explain it. Maybe this proposal also is overengineering because it's a cosmetical issue.

**Review by mborowski511 — APPROVED** [review]
Thanks for those changes; I agree with them.

**Review by ssteinbuss — APPROVED** [review]
Thanks for the effort made. That is a great contribution to the RAM to have the wording on the Metadata Broker harmonized

**Comment by ssteinbuss** [comment]
Editorial changes are to be released immediately. Changelog needs to be updated.

**Pull request merged**